### ISSUE TITLE
Installation problems RPi400

### ISSUE THREAD

**Issue by filip-we** [filip-we/PaperTTY#1]
Misc. issues when installing the repo on Raspberry Pi 400.
- Error compiling c-code for package ` twisted`.

**Comment by filip-we** [comment]
When installing `twisted`  this happens:

```
aarch64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/tmpi1689iao/.venv/include -I/usr/include/python3.11 -c src/twisted/test/raiser.c -o build/temp.linux-aarch64-cpython-311/src/twisted/test/raiser.o
src/twisted/test/raiser.c:198:12: fatal error: longintrepr.h: No such file or directory
  198 |   #include "longintrepr.h"
         |            ^~~~~~~~~~~~~~~
compilation terminated.
error: command '/usr/bin/aarch64-linux-gnu-gcc' failed with exit code 1
```

**Comment by filip-we** [comment]
When pinning the version of `twisted` `poetry lock` complained about Python versions, but that was fixed by dropping support for Python 3.5, see a495fa7890df4d80ab6210073efc0bf86f93253e
